src/sonymn_int.hpp

```diff
@@ -71,9 +74,14 @@ namespace Exiv2 {
         static const TagInfo tagInfo_[];
         static const TagInfo tagInfoCs_[];
         static const TagInfo tagInfoCs2_[];
+        static const TagInfo tagInfoFp_[];
 
     }; // class SonyMakerNote
 
+    DataBuf sonyTagCipher  (uint16_t, const byte*, uint32_t, TiffComponent* const, bool bCipher);
```

Comment:
This function can very well be `static` in the `.cpp`, couldn't it?

Reply:
Yes, you're right, it could be static.   It started of life as a single decipher function and grown.  sonyTagCipher does not need to be visible outside of sonymn_int.cpp.

Reply:
I'm going to update the PR to be sure `sonyTagCipher()` doesn't end up in visible symbols.  I think you'll have to approve that change.  I understand what "squashing commits" does, however I've never done it.

Reply:
```
2335 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ nm -g --demangle lib/libexiv2.0.27.2.1.dylib | grep sonyTag
00000000001c75a0 T Exiv2::Internal::sonyTagCipher(unsigned short, unsigned char const*, unsigned int, Exiv2::Internal::TiffComponent*, bool)
00000000001c77b0 T Exiv2::Internal::sonyTagDecipher(unsigned short, unsigned char const*, unsigned int, Exiv2::Internal::TiffComponent*)
00000000001c7890 T Exiv2::Internal::sonyTagEncipher(unsigned short, unsigned char const*, unsigned int, Exiv2::Internal::TiffComponent*)
2336 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ ce ../src/sonymn_int.cpp 
bbedit "../src/sonymn_int.cpp"

2337 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ make
[ 10%] Built target exiv2-xmp
...
2338 rmills@rmillsmbp:~/gnu/github/exiv2/0.27-maintenance/build $ nm -g --demangle lib/libexiv2.0.27.2.1.dylib | grep sonyTag
00000000001c75a0 T Exiv2::Internal::sonyTagDecipher(unsigned short, unsigned char const*, unsigned int, Exiv2::Internal::TiffComponent*)
00000000001c7890 T Exiv2::Internal::sonyTagEncipher(unsigned short, unsigned char const*, unsigned int, Exiv2::Internal::TiffComponent*)
```

Reply:
Robin, you did not push that commit yet, right?